civihr_employee_portal/src/Forms/AbsenceRequestForm.php

```diff
@@ -418,7 +419,7 @@ protected function duplicate_dates_exist($requestedStartTimestamp, $requestedEnd
         $absencesDataQuery = db_select('absence_list', 'al')
             ->condition('contact_id', $_SESSION['CiviCRM']['userID'])
             ->condition('absence_status', self::CANCELLED_LEAVE_TYPE_ID, '<>')
-            //->condition('absence_end_date_timestamp', strtotime('today'), '>')
+            ->condition('absence_status', self::DENIED_LEAVE_TYPE_ID, '<>')
```

Comment:
In the UI, this status is displayed as "Rejected". To keep consistency with the UI, I think it's better to say "Rejected" instead of "Denied"?  (the ticket title and description also use "Rejected")